stdlib/ext/file-ext.mc

```diff
@@ -54,6 +54,12 @@ external readLine ! : ReadChannel -> (String, Bool)
 let readLine : ReadChannel -> Option String =
   lam rc. match readLine rc with (s, false) then Some s else None ()
 
+-- Reads a given number of bytes from the file.
```

Comment:
What happens if in situations where I read more bytes than that are left? E.g., there are 5 bytes left and I attempt to read 10?

Reply:
The solution to this turned out to be a completely different implementation in the OCaml code. I have added it now!

Reply:
The behavior is that it will return `Some a` where `len(a) == 5`. All calls after will return `None ()`

Reply:
I think the comment could be a bit clearer on this. E.g.
```
Read the provided number of bytes from a `ReadChannel`.
If there is no more content, `None` is returned.
If the number of remaining bytes is smaller than `len`, 
all remaining bytes are returned

```

Reply:
Also, what happens if some error occurs when reading? Is `None` returned or does this raise an error?

Reply:
I've changed it so that it returns `None` if there was an error, or there was an EOF and empty string returned.